pkg/utils/release.go

```diff
@@ -39,11 +38,11 @@ func GetUpgradeReleasesToTest(provider string) (from string, to string, err erro
 	if to == "" {
 		return from, to, nil
 	}
```

Comment:
What about this block?

Reply:
It is just a safeguard in case E2E_RELEASE_VERSION is empty, because it's impossible to calculate a "previous major" version.

Reply:
So what happens in that situation? Does the test error?